EasyCommands/CommandParsers/ParameterProcessors.cs

```diff
@@ -916,26 +916,17 @@ public class WaitProcessor : SimpleCommandProcessor<WaitCommandParameter> {
             }
         }
 
-        public class ControlProcessor : SimpleCommandProcessor<ControlCommandParameter> {
```

Comment:
I'm trying to get rid of SimpleCommandProcessor as I need to make changes to these processors, hence these changes